tests/unit/test_dbt_containerizer.py

```diff
@@ -0,0 +1,32 @@
+import os
+from pathlib import Path
+from palm.containerizer import DbtContainerizer
+
+class MockContext:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+DbtContainerizer.__abstractmethods__ = set()
+
+def test_run(tmp_path, environment):
+    templates_dir = (Path(__file__).parents[2] / 'palm/plugins/core/templates/containerize')
+
+    os.chdir(tmp_path)
+    Path('.env').touch()
+    Path('requirements.txt').touch()
+    ctx = MockContext(obj=environment)
+    c = DbtContainerizer(ctx, templates_dir)
+    c.run()
+
+    assert Path(tmp_path, 'Dockerfile').exists()
+    assert Path(tmp_path, 'requirements.txt').exists()
+    assert Path(tmp_path, 'scripts', 'entrypoint.sh').exists()
+
+def test_validate_dbt_version(tmp_path, environment):
+    ctx = MockContext(obj=environment)
+    default_dbt_version = DbtContainerizer(ctx, tmp_path)
+    assert default_dbt_version.validate_dbt_version()
+
+    valid_dbt_version = DbtContainerizer(ctx, tmp_path, dbt_version='0.19.0')
+    assert valid_dbt_version.validate_dbt_version()
```

Comment:
~~I'd like to see a falsy assertion here, too - to prove we don't support versions below 0.19~~

This is testing a method that does not exist? I think we need to implement this

Reply:
We talked yesterday about removing the version checker (for now) but I forgot to remove the test. I will do that.

Reply:
I think we can still do some version checking at this point, but the VersionChecker class was probably overkill for just this one usecase. I had bigger plans for it... but we need a reliable way to know the dbt_version for it to work. Reading from the dockerfile is one option, I suppose.

Reply:
implemented a basic dbt version checker